### DISCUSSION TITLE
Make Ctrl-Space work in terminal

### DISCUSSION THREAD

**Discussion by carloscabanero** [blinksh/blink#1923] [Q&A] [answered]
How do I enable Ctrl-Space so I can send it as my Tmux prefix? How do I enable Ctrl-Space to be captured on EMacs or Neovim? CTRL+space custom combination mapped on your host, but the same combination not working on Blink?

**Comment by iflowerml** [comment, marked as answer]
Ctrl-Space is taken by iOS for the keyboard language switch, if you want to bypass this in your terminal, you must:

1. Access configuration menu, type command: `config` in Shell OR `CMD` +` ,`  [On Keyboard].
	
![PNG image](https://github.com/blinksh/blink/assets/153754415/54be9c8d-bb70-4025-a3bc-21c44392926a)

2. Go to keyboard > Custom Presses > Add Shortcut. 
	
![PNG image](https://github.com/blinksh/blink/assets/153754415/946a3f9a-4383-4232-bea8-a0121a94d3ac)
![PNG image](https://github.com/blinksh/blink/assets/153754415/1074e3c8-a0e9-49b4-a9fb-b6afdf4bf814)

3. Press your desired combination, `Ctrl`+`Space` or an alternative if you want to remap.
4. In action list choose `^⎵` , as the output sequence to the terminal.
	
![PNG image](https://github.com/blinksh/blink/assets/153754415/1d48ed88-9cd8-4ed3-9059-ad7fcdbda184)
![PNG image](https://github.com/blinksh/blink/assets/153754415/c91efa31-6480-42d0-bceb-2d6177456e23)

Your Custom Press should now work on Blink, try it out!
